employees/views.py

```diff
@@ -569,3 +572,16 @@ def render_to_response(self, context: dict, **response_kwargs: Any) -> HttpRespo
             )
             work_book.save(response)
         return response
+
+
+@method_decorator(login_required, name="dispatch")
+class LoadTaskActivitiesInProjectView(TemplateView):
+    template_name = "employees/partial/task_activity_list.html"
+
+    def get_context_data(self, **kwargs: Any) -> dict:
+        context = super().get_context_data(**kwargs)
+        if "project" not in self.request.GET:
+            raise Http404
+        context["project_id"] = self.request.GET.get("project")
```

Comment:
You don't need this to be passed in context. Also you should validate the value passed from GET, at least if it is an int.